### ISSUE TITLE
anyone having login problem?

### ISSUE THREAD

**Issue by xienxienzai** [Maxx53/SCMBot#552]
Anyone having login problem using multiple bot with same account?
I have this problem at the moment

**Comment by nikita971** [comment]
Yes, all have this problem. Solution: don't use bot.

**Comment by LawrenceCSGO** [comment]
Amazing solution. Maybe I should F5 the market page by myself? :D
BTW, I don't have any problems with multiple bots. I just set up the first instance completely, and use the same files everywhere. Ctrl+C\Ctrl+V style, huh.

**Comment by xienxienzai** [comment]
![untitled](https://cloud.githubusercontent.com/assets/8124151/11550259/00fa432e-99a9-11e5-8f2b-d60d56ef5a22.png)
Hi @LawrenceCSGO ,

Thanks for your reply.Just start on 24hours ago,this thing happenned when i trying to scan items from multiple places using same steam account and i received this error.

**Comment by Alex1os** [comment]
yes this is now normal thx to steam authenication mobile app or steam guard

1 account = 1 bot , u cant even be loged in scm bot and the website on 2 different ip addresses on the same account
